imu_complementary_filter/CMakeLists.txt

```diff
@@ -38,11 +41,11 @@ ament_target_dependencies(complementary_filter geometry_msgs rclcpp message_filt
 
 install(TARGETS
         complementary_filter
-        DESTINATION lib/${PROJECT_NAME}
+        DESTINATION lib
```

Comment:
Change in line 62 is not sufficient. 
Library can be saved in directly in `/workspace/install/imu_complementary_filter/lib/`,
when directory is nested: /workspace/install/imu_complementary_filter/lib/**imu_complementary_filter**/libcomplementary_filter.a
`find_package` can not import the package.

Reply:
Yes, that's definitely a bug. The "default" CMakeLists generated by `ros2 pkg create` also has `DESTINATION lib` here.